src/app/components/ThemeProvider/fontFaces.ts

```diff
@@ -6,8 +6,11 @@ const NOTO_SERIF_SINHALA_FONTS_DIR =
 const NOTO_SANS_TAMIL_FONTS_DIR =
   'https://ws-downloads.files.bbci.co.uk/fonts/NotoSansTamil/v1.00/';
 
-const MALLANA_FONTS_DIR =
-  'https://ws-downloads.files.bbci.co.uk/fonts/Mallanna/v1.0.4/';
+const NOTO_SANS_TELUGU_FONTS_DIR =
+  'https://ws-downloads.files.bbci.co.uk/fonts/NotoSansTelugu/v1.00/';
+
+const NOTO_SANS_GUJARATI_FONTS_DIR =
+  'https://ws-downloads.files.bbci.co.uk/fonts/NotoSansGujarati/v1.00/';
```

Comment:
Not live yet, hence 'do not merge' label

Reply:
is there a PR for the font server repo or is that already up there/has it been merged?

Reply:
Yep it's linked in this ticket: https://jira.dev.bbc.co.uk/browse/WSTEAM1-1003

It has been merged but I'm still working through the steps to deploy